catsgo.py

```diff
@@ -6,6 +6,7 @@
 import time
 import sys
 import logging
+import traceback
```

Comment:
Is this still used?

Reply:
Yes, I need the traceback imported so I can have good description of exceptions